src/render/boolean.ts

```diff
@@ -1,6 +1,15 @@
-export function isTruthy (val: any): boolean {
-  return !isFalsy(val)
+import { Context } from '../context/context'
+
+export function isTruthy (val: any, ctx: Context): boolean {
+  if ((val === null) && ctx && ctx.opts && ctx.opts.jsTruthy ) return false;
```

Comment:
To achieve completeness and correctness, it's better to directly return `!isFalsy(val, ctx)`.